yjit/src/backend/arm64/mod.rs

```diff
@@ -153,7 +153,9 @@ impl Assembler
                         out_opnd
                     }
                 },
-                _ => asm.load(opnd)
+                _ => {
+                    asm.load(opnd)
+                }
```

Comment:
```suggestion
                _ => asm.load(opnd)
```

It was an unneeded diff made during debugging